setup.py

```diff
@@ -5,7 +5,7 @@
 from setuptools import setup
 
 
-version = '1.0.2'
+version = '1.1.0'
```

Comment:
Why not 1.0.3 ?

Reply:
dropping older Django versions and adding python 3.8 seemed a significant change.